packages/server/lib/browsers/index.js

```diff
@@ -134,7 +136,9 @@ module.exports = {
     }
 
     // Instance will be null when we're dealing with electron. In that case we don't need a browserCriClient
-    return browserLauncher.connectToNewSpec(browser, options, automation)
+    await browserLauncher.connectToNewSpec(browser, options, automation)
+
+    return this.getBrowserInstance()
```

Comment:
These functions are called as part of the browser launch sequence in certain scenarios; I updated them to return the browser instance so that we can get a consistent value resolved from `open_project`s `launch` function.